client-cli/src/command/address_command.rs

```diff
@@ -146,7 +146,10 @@ impl AddressCommand {
         let wallet_kind = wallet_client.get_wallet_kind(name, &enckey)?;
         if wallet_kind == WalletKind::HW {
             let hw_kind = ask_hardware_kind(None)?;
-            wallet_client.update_hw_service(hw_kind);
+            wallet_client.update_hw_service(hw_kind).map_err(|e| {
+                error("maybe need sudo user");
```

Comment:
maybe better to point to this link: https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues ?

Reply:
`sudo` is only needed for updating the udev rules